src/include/migraphx/verify.hpp

```diff
@@ -187,16 +188,95 @@ double rms_range(const R1& r1, const R2& r2)
         return std::numeric_limits<range_value<R1>>::max();
 }
 
+template <class R>
+double get_rms_tol(const R&, std::size_t tolerance = 80)
+{
+    double threshold = std::numeric_limits<range_value<R>>::epsilon() * tolerance;
+    return threshold;
+}
+
+/*
+C++ doesn't support named arguments, this is just wrapper that helps distinguish between actual
```

Comment:
No reason to guard to distinguish between results and expected arguments:
<Ideally, a design should have symmetry and if two arguments (i.e. results vs. expected) are reversed, chalk it up to programming or user error. Yet, the outcome of that would be still logical -- example: in diffing two files, their order produces different results, but still logical. Maybe some difference here would reverse polarity.>

Reply:
It can't be symmetric for the elementwise verification 
https://numpy.org/doc/stable/reference/generated/numpy.allclose.html

allclose calculates `rtol *b`.  If it is `rtol*a` it can result in different value.

Reply:
Design symmetry (of an API) doesn't mean the results are identical if inputs order is changed. As I mentioned above in the example about diffing two files. No body should expect identical results due to symmetry in design -- when input order is changed. Thanks.

Reply:
> if two arguments (i.e. results vs. expected) are reversed, chalk it up to programming or user error.

We want it to do the right thing. Chalking it up to progammer error just leads to subtle bugs in our code, since the flipped arguments are not diagnosed.

Reply:
Accuracy testing is a science. We are mixing it up with proper API design. 
Anyways, I have belabored it enough :-)